core/src/main/scripts/importer/validateData.py

```diff
@@ -2138,11 +2145,43 @@ class FusionValidator(Validator):
         'Method',
         'Frame']
     REQUIRE_COLUMN_ORDER = True
+    ALLOW_BLANKS = True
+
+    def __init__(self, *args, ** kwargs):
+        super(FusionValidator, self).__init__(*args, **kwargs)
+        self.fusion_lines = {}
 
     def checkLine(self, data):
+
         super(FusionValidator, self).checkLine(data)
-        # TODO check the values
 
+        # parse hugo and entrez to validate them together
```

Comment:
It's slightly different because in mutationi event, Entrez Gene ID can be 0 and Hugo_Symbol Unkown. We discussed it and decided to add this difference but not combine the 2 code blocks now. This code is also used in other locations and refactoring it will take result in a lot of changes, out of the scope of this PR.